src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -1519,6 +1519,39 @@ public void setOriginalSequenceNumber(int sequenceNumber)
     }
 
     /**
+     * Sets the padding length for this RTP packet.
+     *
+     * @param len the padding length.
+     * @return the number of bytes that were written, or -1 in case of an error.
+     */
+    public int setPaddingSize(int len)
```

Comment:
setPaddingSize changes two bytes in different places in the packet. I don't see how anyone can use that to increment an offset. Also, I don't see how this is consistent with the other methods, since all other setters (sequenceNumber, timestamp, marker, extensionBit, payloadType, SSRC, originalSequenceNumber) are void.